Error at install following dedicated web server instructions

### Issue details

When following [these instructions](http://doc.wallabag.org/en/master/user/installation.html#on-a-dedicated-web-server-recommended-way), I have this error when I `make install`:

`HEAD is now at 501a4da... Release wallabag 2.1.2
scripts/install.sh: 9: scripts/install.sh: composer: not found
PHP Warning:  require(/home/thomas/wallabag/app/../vendor/autoload.php): failed to open stream: No such file or directory in /home/thomas/wallabag/app/autoload.php on line 9
PHP Fatal error:  require(): Failed opening required '/home/thomas/wallabag/app/../vendor/autoload.php' (include_path='.:/usr/share/php') in /home/thomas/wallabag/app/autoload.php on line 9
Makefile:15: recipe for target 'install' failed
make: *** [install] Error 255`

Indeed, there is no vendor folder in the git repository as indicated [here](https://github.com/wallabag/wallabag/blob/master/app/autoload.php#L9)

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.1.2
* How did you install wallabag? Via `git clone`
* php version: 7
* OS: Ubuntu 16.04
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): PostgreSQL

### Steps to reproduce/test case

Clone the repo, `cd wallabag && make install`

--- a/Makefile
+++ b/Makefile
@@ -41,6 +41,6 @@ travis: ## Make some stuff for Travis-CI
 deploy: ## Deploy wallabag
 	@bundle exec cap staging deploy
 
-.PHONY: help clean install update build test release travis deploy
+.PHONY: help clean install update build test release travis deploy run dev
 
 .DEFAULT_GOAL := install

--- a/scripts/install.sh
+++ b/scripts/install.sh
@@ -2,6 +2,8 @@
 # You can execute this file to install wallabag
 # eg: `sh install.sh prod`
 
+command -v composer >/dev/null 2>&1 || { echo >&2 "wallabag requires composer but it's not installed (see http://doc.wallabag.org/en/master/user/installation.html). Aborting."; exit 1; }
+
 ENV=$1
 TAG=$(git describe --tags $(git rev-list --tags --max-count=1))
 

